### ISSUE TITLE
Sympa Accessibility Issue - 18 - In Mobile View, submenus are not accessible by keyboard

### ISSUE THREAD

**Issue by rm1-uiuc** [sympa-community/sympa#1749]
The University of Illinois at Urbana-Champaign, Technology Services Accessibility Testing team conducted an Accessibility Review of Sympa on September 25, 2023 and found 26 issues that need to be fixed to make the interface accessible for visual impaired users.

Version
-------
6.2.72

Installation method
-------------------
Source Package

Expected behavior
-----------------
When a user reduces the Sympa interface screen in a web browser the submenus should still be accessible by keyboard.   The screen should be dynamic and adjust with the resolution.

Actual behavior
---------------
When the screen is reduced to a mobile view, the submenus are not accessible by keyboard. Back button on Basic Operations menu is also not keyboard accessible.

Steps to reproduce
------------------
Sympa Accessibility Issue -18 was determined by manual functional testing performed by the user navigating with keyboard only or using assistive technology such as screen reader. See attached review for more info on specific location in interface.

When the screen is reduced to a mobile view, the submenus are not accessible by keyboard. Back button on Basic Operations menu is also not keyboard accessible.

[WCAG Level A 2.1.1 Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html)

![image](https://github.com/sympa-community/sympa/assets/144282683/add0240d-0add-44e1-ac09-88b9c1c039b4)

Fig 20 Keyboard accessibility issue.

![image](https://github.com/sympa-community/sympa/assets/144282683/a2375575-c4ac-4dd7-b68b-d38a2fe42f60)

Fig 21 Keyboard accessibility issue on menu .

Additional information
----------------------
Attached is the Accessibility Review conducted on Sept 25,2023
[Tech Services Review - Accessibility Evaluation Report for Sympa.pdf](https://github.com/sympa-community/sympa/files/13505199/Tech.Services.Review.-.Accessibility.Evaluation.Report.for.Sympa.pdf)

**Comment by ikedas** [comment]
> When the screen is reduced to a mobile view, the submenus are not accessible by keyboard.

This looks a duplicate of "Issue 19" (#1750).

>  Back button on Basic Operations menu is also not keyboard accessible.

This is a similar issue as above: Appropriate keyboard action such as typing Enter key allows go back from submenu.   Thus I think  the Back button is keyboard accessible.

**Comment by ikedas** [comment]
Closed as a duplicate of #1750.
